tests/testthat/test-visr_survfit.R

```diff
@@ -47,6 +47,8 @@
 #' T5. The final object is a ggplot of class `ggsurvfit`.
 #' T5.1 The final object is a ggplot of class `ggplot`.
 #' T5.2 The final object is a ggplot of class `ggsurvfit`.
+#' T6. The final object does not exclude parts of KM estimate.
```

Comment:
Because it is vague, I dont know what you are testing:
_The final object does not exclude parts of KM estimate._ Which parts? and what is the purpose? Did you mean that the object will display all available time points even if the x axis argument is used?

Reply:
This entire PR makes one small change, and from my POV it seems like a bad-faith argument that it's unclear what is intended to be tested here (even if there were no text describing it). If you would like something else here, go ahead and update it.